content/techniques/mvc.md

```diff
@@ -94,6 +94,42 @@ In this code, we are specifying the template to use in the `@Render()` decorator
 
 While the application is running, open your browser and navigate to `http://localhost:3000`. You should see the `Hello world!` message.
 
+#### Aliasing Routes
+
+You can alias routes with the `@WithAlias()` decorator in your controller, and then reference the alias inside your view.
+
+Suppose in your user controller, you have some routes to a list of users or to a specific user:
+```typescript
+@@filename(users.controller)
+import { Get, Controller, Render, WithAlias } from '@nestjs/common';
+import { User } from './user.entity.ts';
+
+@Controller('users')
+export class UserController {
+  @Get()
+  @Render('users-list')
+  @WithAlias('hello')
+  root(): User[] {
+    return [ /* list of users */ ];
+  }
+  
+  @Get('/:user')
+  @Render('user')
+  @WithAlias('hello')
```

Comment:
Are these two routes supposed to have the same alias?

Reply:
@jmcdo29 , whoops. Nope. Correcting...

Reply:
@jmcdo29 , should be updated now.